GrandCentralBoard/Core/Models/Source.swift

```diff
@@ -17,15 +17,28 @@ public protocol UpdatingSource : class {
 
 public protocol Source : UpdatingSource {
 
-    typealias ResultType
+    associatedtype ResultType
 
     var sourceType: SourceType { get }
 }
 
+/**
+ The source will return value synchronously in a non-blocking way.
+ */
 public protocol Synchronous : Source {
     func read() -> ResultType
 }
 
+/**
+ The source will call the provided block after the value is retrieved (only once).
+ */
 public protocol Asynchronous : Source {
     func read(closure: (ResultType) -> Void)
+}
+
+/**
+ The source will call the provided block each time a new value arrives (multiple times). Note that `optimalFrequency` can and often will be ignored.
```

Comment:
What does the `optimalFrequency` refer to? Isn't this supposed to be `interval`?